tests/test_lm/test_integration/test_jsrun/test_3/res_set

```diff
@@ -0,0 +1,2 @@
+cpu_index_using: physical
+rank: 0: { host: 1; cpu: {0,4,8,12,16,20,24,28}}
```

Comment:
Could this also be:
`rank: 0: { host: 1; cpu: {1,2,3,4,5,6,7,8}}`

Reply:
No, unfortunately it is not. Regardless the index we use, we have to refer to cores based on their logical ID. This was communicated to me by ORNL, when we were discussing the SMT levels. 
Quote from a ticket with ORNL:
>  This seems a little misleading, as I'd expect "physical" to accept numbers 0-41 and virtual to take 0-167 (when in SMT4 mode). Instead, in SMT1 it seems like the 0-167 designation remains but only the "first" slice on each core is honored. I'll have to check into that.

As a result we need to keep the logical numbering until ORNL gets back at me.